api/src/routing.js

```diff
@@ -302,8 +302,8 @@ app.get(routePrefix + 'login/token/:token?', login.tokenGet);
 app.postJson(routePrefix + 'login/token/:token?', login.tokenPost);
 app.get(routePrefix + 'password-reset', login.getPasswordReset);
 app.postJson(routePrefix + 'password-reset', login.resetPassword);
-app.get(routePrefix + 'login/oidc', login.oidcAuthorize);
-app.get(routePrefix + 'login/oidc/get_token', login.oidcLogin);
+app.get(routePrefix + 'login/oidc/authorize', login.oidcAuthorize);
+app.get(routePrefix + 'login/oidc', login.oidcLogin);
```

Comment:
Happy to discuss this more (since this endpoints are a big deal), but the longer I looked at this, the more I liked the naming you had for the controller functions (and the more I felt like those names did not match the endpoint names).  

I like `login/oidc` doing the actual login action where you get the session cookie.  (Technically, if you had a valid auth token you could call this endpoint directly (out-of-flow) with the token and get yourself logged in.)

Reply:
LGTM!